cast/src/launcher/layout/hc-connect.ts

```diff
@@ -1,10 +1,18 @@
-import "@material/mwc-button";
-import { mdiCastConnected, mdiCast } from "@mdi/js";
+import type { CastManager } from "../../../../src/cast/cast_manager";
 import type {
   Auth,
   Connection,
   getAuthOptions,
 } from "home-assistant-js-websocket";
+import type { TemplateResult } from "lit";
+
+import "../../../../src/components/ha-svg-icon";
+import "../../../../src/components/ha-textfield";
+import "../../../../src/layouts/hass-loading-screen";
+import "./hc-layout";
+import "@material/mwc-button";
```

Comment:
I dont agree with this sorting, external packages used to be on the top, and internal ones at the bottom

Reply:
I agree, but I would like to have side-effect imports seperated because that are most of the times our lit elements. But doing this and have external side-effects on the top is not possible with this and with no other plugin :unamused:

Reply:
Some people seems to be used to the current sorting. 😆 

Reading about this specific rule, the perfectionist/sort-import has a customGroups option: https://perfectionist.dev/rules/sort-imports#customgroups. I don't think it will work for what you are trying to accomplish, but I may note it here for in case you have missed it. 

Apart from all the things about the sorting, are there are technical reasons to apply the import order in this way?

Reply:
Thanks, I can try, but I don't think it works in that detail.

No technical reasons, I just suggest it to be discussed.

But I am not happy on how the sorting works even without the side-effects rule. I think I'll close the PR